### DISCUSSION TITLE
Not using Elixir?

### DISCUSSION THREAD

**Discussion by munjalpatel** [wasmCloud/wasmCloud#2393] [Q&A] [answered]
I thought wasmCloud was built on Elixir. All I see is Rust here -- nothing against Rust though!
https://www.youtube.com/watch?v=l0QS_iXlQto

**Comment by vados-cosmonic** [comment, marked as answer]
Hey @munjalpatel preivous versions of wasmCloud *were* built on Elixir, and called out to Rust -- This all changed in the [0.78.0 release](https://wasmcloud.com/blog/wasmcloud-0.78.0-componentized-rust), and we've got a bunch written about it there!
